internal/cri/config/config.go

```diff
@@ -320,6 +321,8 @@ type ImageConfig struct {
 	// TODO: Migrate to transfer service
 	MaxConcurrentDownloads int `toml:"max_concurrent_downloads" json:"maxConcurrentDownloads"`
 
+	ConcurrentDownloadLimiter *semaphore.Weighted
```

Comment:
This doesn't seem right. The cri config is supposed to be serializable.

Reply:
Maybe we can find a better home for this. It seems to just be a holding spot that's created from the value passed for `MaxConcurrentDownloads`

Reply:
Good point, I moved it up one struct ! LMK !